specs/form-associated-custom-element.md

```diff
@@ -0,0 +1,114 @@
+# Form Associated Custom Elements
+
+## Overview
+This spec regards a general implementation to facilitate form communication and connection for components that should be form-associated, including checkboxes, radios, text inputs, and other components which store user input and should be captured during form submission.
```

Comment:
Wording seems a little tough to understand, maybe just "This spec defines a general implementation of form-associated components with special attention paid to form specific behaviors."